modules/shared/partials/emergency-shadowing-callout.adoc

```diff
@@ -1,5 +1,5 @@
 [IMPORTANT]
 .Experiencing an active disaster?
 ====
-See xref:deploy:redpanda/manual/resilience/shadowing-guide.adoc[Emergency Shadowing Failover] for immediate step-by-step emergency procedures.
+See xref:deploy:redpanda/manual/disaster-recovery/shadowing/disaster-response.adoc[Disaster response procedures] for immediate step-by-step disaster procedures.
```

Comment:
```suggestion
See xref:deploy:redpanda/manual/disaster-recovery/shadowing/disaster-response.adoc[] for immediate step-by-step disaster procedures.
```
suggest doing this for all the titles, since they may change